code/game/objects/items/natural.dm

```diff
@@ -299,131 +299,155 @@
 
-		/datum/attunement/earth = -0.2,
+		/datum/attunement/fire = -0.15,
 	)
 
 /obj/item/natural/elementalfragment
 	name = "elemental fragment"
 	icon_state = "fragment"
-	desc = "A mystical essence embued with the power of Dendor. Merely holding it transports one's mind to ancient times."
+	desc = "A small- fractured piece of a Behemoth's core. Don't hold this for longer than you have to."
 	resistance_flags = FIRE_PROOF
 	w_class = WEIGHT_CLASS_SMALL
 	sellprice = 20
 
 	attunement_values = list(
-		/datum/attunement/electric = 0.1,
-		/datum/attunement/ice = 0.1,
-		/datum/attunement/blood = 0.1,
-		/datum/attunement/aeromancy = 0.1,
+		/datum/attunement/electric = 1,
+		/datum/attunement/ice = 1,
+		/datum/attunement/aeromancy = 1,
 
-		/datum/attunement/earth = -0.15,
+		/datum/attunement/earth = -0.5,
 	)
 
 /obj/item/natural/elementalrelic
 	name = "elemental relic"
 	icon_state = "relic"
-	desc = "A mystical essence embued with the power of Dendor. Merely holding it transports one's mind to ancient times."
+	desc = "One of three ley-keys. \nA massive, jagged rock that reminds you of the scale of the world you're dealing with. It crackles and vibrates with power just barely yours."
 	resistance_flags = FIRE_PROOF
 	w_class = WEIGHT_CLASS_SMALL
 	sellprice = 20
 
 	attunement_values = list(
-		/datum/attunement/electric = 0.1,
-		/datum/attunement/ice = 0.1,
-		/datum/attunement/blood = 0.1,
-		/datum/attunement/aeromancy = 0.1,
+		/datum/attunement/electric = 3,
+		/datum/attunement/ice = 3,
+		/datum/attunement/aeromancy = 3,
 
-		/datum/attunement/earth = -0.1,
+		/datum/attunement/earth = -1.5,
 	)
 
 //Nullmagic
 /obj/item/natural/voidstone
 	name = "voidstone"
 	icon_state = "voidstone"
-	desc = "An incredibly rare substance torn from creatures immune to magick. This material forsakes Noc's gifts."
+	desc = "An incredibly rare substance torn from creatures immune to magick. \nThis material forsakes Noc's gifts- its use could only be described as deeply heretical."
 	resistance_flags = FIRE_PROOF
 	w_class = WEIGHT_CLASS_SMALL
 	sellprice = 20
 
+	attunement_values = list( //Should be the antithesis of Noc's attunements. Should primarily be useful to liches and other dark mages.
+		/datum/attunement/arcyne = -2,
+		/datum/attunement/ice = -2,
+		/datum/attunement/dark = 1,
+		/datum/attunement/death = 1,
+		/datum/attunement/blood = 1,
+
+	)
+
+/obj/item/natural/melded/t5
+	name = "arcanic aberation"
+	icon_state = "wessence"
+	desc = "A melding of arcane fusion and voidstone. It pulses erratically, power coiled tightly within and dangerous. This object scorns Noc's gifts. Many would be afraid of going near this, let alone holding it. "
+
+	attunement_values = list( //Might be more interesting for the introduction of a voidstone to increase attunement in the opposite direction. Again, another item that liches and dark(er) mages should LOVE.
+		/datum/attunement/arcyne = -2,
+		/datum/attunement/ice = -2,
+		/datum/attunement/dark = 6,
```

Comment:
I get this is high tier but every single item gives this much attunement to the mob that carries it, which 
1 maxes out your attunement because the max is 2.5 
2 is 5x the apex attunement here
3 is strictly negative to have more than one

Even the high tier stuff shouldn't go above 1 is my take on this